.github/workflows/cicd-1-pull-request.yaml

```diff
@@ -27,11 +49,25 @@ jobs:
     steps:
       - name: "Checkout code"
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: "Set up Node.js"
+        uses: actions/setup-node@v4
+        with:
+          node-version: 16
+          cache: npm
+      - name: "Install dependencies"
+        run: npm ci
+      - name: "Build action bundle (dry-run context)"
+        run: npm run package
       - name: "Semantic Release"
         uses: cycjimmy/semantic-release-action@v4.1.1
```

Comment:
Something for later - we should consider whether forking this GitHub Action into our organisation would be beneficial if we plan to adopt this process more widely, as it ultimately "handles" the deployment of our code to live/prod